examples/frameworks/microsoft_agent_framework_demo/src/microsoft_agent_framework_demo/hotel_price_tool.py

```
@@ -0,0 +1,81 @@
+    checkin: str
+    checkout: str
+
+
+class HotelOffersResponse(BaseModel):
+    offers: list[HotelOffer]
+
+from agent_framework import ai_function
+
+@ai_function(name="hotel_price", description="Retrieves hotel prices information for any location")
+@register_function(config_type=HotelPriceToolConfig)
+async def hotel_price(tool_config: HotelPriceToolConfig, builder: Builder):
```

Ruff (0.14.7)

Unused function argument: `builder`

(ARG001)